### ISSUE TITLE
Benchmark datasets

### ISSUE THREAD

**Issue by trappmartin** [stevie1023/cspn-gpspn#4]
It would be good to additional benchmark on the datasets used in: https://papers.nips.cc/paper/7905-heterogeneous-multi-output-gaussian-process-prediction.pdf

Those are heterogeneous datasets, so even more interesting. I'll open an additional issue explaining how to work with heterogeneous data.

**Comment by trappmartin** [comment]
Maybe also check the UCI repository for data: https://archive.ics.uci.edu/ml/datasets.php?format=&task=reg&att=num&area=&numAtt=&numIns=&type=mvar&sort=nameUp&view=table

**Comment by minimrbanana** [comment]
And the following Multitask_Exact_GPs are the one I mentioned, not sure if it fits well.
https://github.com/cornellius-gp/gpytorch/tree/master/examples/03_Multitask_Exact_GPs

**Comment by trappmartin** [comment]
Yes, that looks like a good reference. They also describe two datasets that we can use.

**Comment by stevie1023** [comment]
Hi, Martin! Hope you're safe and sound under the current corona situation~ I have another question though- - After I had the model with 4 children trained for usflight dataset, I come across this out-of-memory problem again while doing testing even though I tried using batches. Do you think it's because of the model structure or I simply did  something wrong in testing?
